packages/mdc-ripple/util.js

```diff
@@ -34,7 +42,32 @@ export function supportsCssVariables(windowObj) {
     windowObj.CSS.supports('(--css-vars: yes)') &&
     windowObj.CSS.supports('color', '#00000000')
   );
-  return explicitlySupportsCssVars || weAreFeatureDetectingSafari10plus;
+
+  if (!explicitlySupportsCssVars && !weAreFeatureDetectingSafari10plus) {
+    supportsCssVariables_ = false;
+    return supportsCssVariables_;
+  }
+
+  // Detect versions of Edge with buggy var() support
```

Comment:
Put this whole section below in a separate function, which will make it easier to document the various arguments